src/components/player/cue-bar.tsx

```diff
@@ -72,10 +95,16 @@ const CuePopup: React.FC<any> = ({cue, active}) => {
   ) : null
 }
 
-const NoteCue: React.FC<any> = ({cue, duration, className}) => {
-  const [active, setActive] = useCue(cue)
+const NoteCue: React.FC<any> = ({
+  cue,
+  duration,
+  className,
+  actions,
+  player,
+}) => {
+  const setActive = useCue(cue, actions)
   const [persist, setPersist] = React.useState(false)
-
+  const active = cue === player.activeMetadataTrackCue
```

Comment:
moved the active cue to the player state (requires cueplayer version `yarn install`)